### ISSUE TITLE
Increase font size

### ISSUE THREAD

**Issue by eunovm** [KillPinguin/grep-linux-patch-statistics#1]
Generate pictures have enough space for the increased font size. After that it will be much easier to read.

**Comment by klockeph** [comment]
You mean for the labels etc.?
Feel free to create a pull request! :+1:

**Comment by eunovm** [comment]
Yes, exactly. I tried to follow your README, but I could not generate figures after all. It seems that there is no tool specific information within your scripts. If you have a working setup, likely it will be very easy to you to implement this issue.

**Comment by klockeph** [comment]
Unfortunately I stopped working on this project and do no longer have a working setup :/
